django/utils/text.py

```diff
@@ -82,7 +82,7 @@ def chars(self, num, truncate=None, html=False):
 
         `truncate` specifies what should be used to notify that the string has
         been truncated, defaulting to a translatable string of an ellipsis
-        (...).
+        (…).
```

Comment:
I think you could remove the example of an ellipsis in the docstring here and below.